.teamcity/settings.kts

```diff
@@ -1088,3 +1062,100 @@ object WpDesktop_DesktopE2ETests : BuildType({
 	}
 })
 
+object WPComPlugins : Project({
+    name = "WPCom Plugins"
+    buildType(WPComPlugins_EditorToolKit)
+    params {
+        text("docker_image_wpcom", "registry.a8c.com/calypso/ci-wpcom:latest", label = "Docker image", description = "Docker image to use for the run", allowEmpty = true)
+    }
+})
+
+object WPComPlugins_EditorToolKit : BuildType({
+    name = "Editor ToolKit"
+
+    artifactRules = "editing-toolkit.zip"
+
+    vcs {
+        root(WpCalypso)
+        cleanCheckout = true
+    }
+
+    steps {
+        script {
+            name = "Prepare environment"
+            scriptContent = """
+                #!/bin/bash
+
+                # Update node
+                . "${'$'}NVM_DIR/nvm.sh" --no-use
+                nvm install
+
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+
+                # Update composer
+                composer install
+
+                # Install modules
+                yarn install
+            """.trimIndent()
+            dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
+            dockerPull = true
+            dockerImage = "%docker_image_wpcom%"
+            dockerRunParameters = "-u %env.UID%"
+        }
+        script {
+            name = "Run tests"
+            executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
+            scriptContent = """
+                #!/bin/bash
+
+                # Update node
```

Comment:
Interesting, is this not shared with the previous step?

Reply:
It is not :(

Each build step runs in a different shell process, so all env vars (and ultimately, `nvm` is setting up `PATH`) are fresh and need to be set up again.

The good news is this is cumbersome to write, but execution is fast: most of the time it just uses the node version included in the base image and this command completes in a couple of seconds.